include/glow/PassManager/PassManager.h

```diff
@@ -149,6 +154,12 @@ class PassManagerBase : public Named {
   virtual std::atomic<unsigned> &globalPassCounter() = 0;
   /// Get the global pass counter associated with the pass manager.
   virtual const std::atomic<unsigned> &globalPassCounter() const = 0;
+
+  /// increase the index and return the current index
+  int globalInterationCounter(){
+    iterationCount++;
```

Comment:
If we keep this function, I'd prefer either the caller incrementing this (by returning `iterationCount` by reference like `globalPassCounter()` does) or renaming this function to `incrementAndGetIterationCount()` or something similar.

Reply:
Thanks. I removed this function.